docs/sdk/auth/fragments/android/working-with-api.md

```diff
@@ -118,7 +118,27 @@ AWSMobileClient.getInstance().signIn(username, password, null, new Callback<Sign
 });
 ```
 
-## Confirm SignIn
+### Confirm SignIn (MFA)
+
+In order to setup multi-factor authentication, choose `Manual configuration` while setting up auth in the CLI and when you get to the `Multifactor authentication` step, choose these values:
+
+```bash
+ Multifactor authentication (MFA) user login options: ON (Required for all logins, can not be enabled later)
```

Comment:
Nit for the CLI: you stylized this as multi-factor Authentication above, which is how AWS refers to it elsewhere. We should consider changing for consistency.